packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -50,8 +51,9 @@ class CupertinoPageScaffold extends StatelessWidget {
     if (navigationBar != null) {
       final MediaQueryData existingMediaQuery = MediaQuery.of(context);
 
-      // TODO(https://github.com/flutter/flutter/issues/12912):
+      // TODO(xster):
```

Comment:
nit: you can remove the newline after the colon, too. that way grepping for "TODO" will show the message